code/src/artists.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import data from './data.json';
+
+export const artists = () => {
+    return (
+        <div>
+            <data />
```

Comment:
This wouldn't work (React has no idea how to render json), but it looks like this component isn't being used anyway, so it should be removed! Less code is always better! :)